src/opnsense/mvc/app/views/OPNsense/Diagnostics/traffic.volt

```diff
@@ -98,15 +98,15 @@ POSSIBILITY OF SUCH DAMAGE.
                       maintainAspectRatio: false,
                       scales: {
                           xAxes: [{
-                              type: 'realtime',
                               time: {
+                                  tooltipFormat:'HH:mm:ss',
                                   unit: 'second',
-                                  minUnit: 'seconds',
+                                  minUnit: 'second',
                                   displayFormats: {
                                       second: 'HH:mm:ss',
-                                      minute: 'HH:mm:ss'
```

Comment:
@kulikov-a the `minute`  option is needed to prevent round minutes to being formatted otherwise.

Reply:
exactly! thanks! missed this (